components/glow.lua

```diff
@@ -371,5 +372,38 @@ binder:SetScript("OnEvent", function()
         end
     end
 
+    -- Support for AzeriteUI5's LibActionButton
+    if (LAB_GE) then
+        local buttonUpdateFunc = function(lib, event, self)

```

Comment:
I'm not fond of redefining a function that overrides existing behavior with slight changes. If the previous `buttonUpdateFunc` is updated, this one must be updated as well. I also believe it is confusing if someone else wants to add (yet another) lib action button later on.

I think the same effect can be achieved with this code (not tested yet):
```lua
-- Support for AzeriteUI5's LibActionButton
if (LAB_GE) then
    -- Glow functions crafted for this library
    local libGlowForGE = {
        ShowOverlayGlow = function(button)
            button:SetSpellActivationColor(1, 1, 1);
            button:ShowSpellActivation();
        end,
        HideOverlayGlow = function(button)
            button:HideSpellActivation();
        end
    }
    
    local LAB_GEButtonUpdateFunc = function(event, self)
        buttonUpdateFunc(libGlowForGE, event, self);
    end
end
```

Reply:
I embedded those glow methods into LAB_GE instead, figured that makes even more sense. So there's no longer any need for a modified buttonUpdateFunc or a proxy/dummy library.

Reply:
> I embedded those glow methods into LAB_GE instead

What do you mean by that?

Have you modified the code of the AzeriteUI addon itself to define a `ShowOverlayGlow` and `HideOverlayGlow` methods?

Reply:
I tested it. Although the glowing button is less flashy, it works, as intended by AzeriteUI, I guess.